webapp/src/test/java/org/apache/falcon/resource/EntityManagerJerseyIT.java

```diff
@@ -198,6 +198,8 @@ public void testOptionalInput() throws Exception {
         File tmpFile = TestContext.getTempFile();
         EntityType.PROCESS.getMarshaller().marshal(process, tmpFile);
         schedule(context);
+        waitForStatus(EntityType.PROCESS.name(), context.getProcessName(), START_INSTANCE,
```

Comment:
I think it will be useful to document the behavior if some instances in the input range for an optional input are available explicitly.